docs/guides/BUILD.md

```diff
@@ -199,6 +199,16 @@ After running `gulp server` to start a *LiveReload* server in your project root:
 * Open Dev Tools and debug...
 
 
+### <a name="devcontainer"></a> VS Code DevContainer
+
+When using [VS Code](https://code.visualstudio.com/) as a development environment, you can [develop inside a container](https://code.visualstudio.com/docs/remote/containers) to allow for a predefined environment in which AngularJS Material code is being built.
+
+To use the DevContainer environment defined for AngularJS Material, just open the project in VS Code and you should be prompted to use the container automatically, if you have [Remote Development extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) installed.
+
+When you first try to open the project in the container, VS Code will build the container (this can take a while) and then make our source code available to it. When we build the documentation in the container and serve it through `gulp site`, the port will automatically be forwarded and the documentation will be available on your local development machine on http://localhost:8080.
```

Comment:
```suggestion
When you first try to open the project in the container, VS Code will build the container (this can take a while) and then make our source code available to it. When we build the documentation in the container and serve it through `npm run docs:watch`, the port will automatically be forwarded and the documentation will be available on your local development machine on http://localhost:8080.
```

Reply:
We don't really want to refer directly to `gulp` commands anymore. I'll open a PR soon to clean up this guide.

Reply:
PR here: https://github.com/angular/material/pull/11880